src/components/Book.js

```diff
@@ -1,22 +1,30 @@
+/* eslint-disable import/extensions */
 /* eslint-disable react/prop-types */
-/* eslint-disable react/destructuring-assignment */
 
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { RemoveBook } from '../Redux/Books/Books.js';
```

Comment:
- [ ] Kindly fix the ESLint error you're having on prop-types. The ESLint check is there to help make you write better and more professional code like real developers do. Disabling these checks defies its purpose. Please fix them.
- [ ] Also, please research for every linter check you have and fix it instead of disabling them 👍 

**See Hint on how to use prop-types from the React Docs**
```mdx
import PropTypes from 'prop-types'

function HelloWorldComponent({ name }) {
  return (
    <div>Hello, {name}</div>
  )
}

HelloWorldComponent.propTypes = {
  name: PropTypes.string
}

export default HelloWorldComponent
```
[Check the official docs ](https://reactjs.org/docs/typechecking-with-proptypes.html)